core/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -357,7 +357,11 @@ object RDDCheckHelper {
 
 object ExecHelper {
   private val UDFRegExLookup = Set(
-    ".*UDF.*".r
+    ".*(?<!python)UDF.*".r
+  )
+
+  private val pythonUDFRegExLookUp = Set(
```

Comment:
Same here as my previous comment. There are three regular expressions that look overlapping.